src/components/organisms/Header.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBook, faBars } from '@fortawesome/free-solid-svg-icons';
+import UserAuthBtn from '../molecules/UserAuthBtn';
+
+function Header() {
+  const [collapse, setCollapse] = useState();
+  const [toggleClass, setToggleClass] = useState('');
+  const [toggleAria, setToggleAria] = useState('');
```

Comment:
Get rid of the aria attribute. You really don't need it unless you're doing a ton of extra work for screen readers.